camayoc/tests/qcs/api/v1/scans/test_vcenter_scans.py

```diff
@@ -0,0 +1,137 @@
+# coding=utf-8
+"""Tests for ``Scan`` API endpoint for quipucords server.
+
+:caseautomation: automated
+:casecomponent: api
+:caseimportance: high
+:caselevel: integration
+:requirement: Sonar
+:testtype: functional
+:upstream: yes
+"""
+
+import pytest
+
+from camayoc import config
+from camayoc.exceptions import ConfigFileNotFoundError
+from camayoc.qcs_models import (
+    Credential,
+    Source,
+    Scan
+)
+from camayoc.tests.qcs.api.v1.scans.utils import wait_until_state
+
+
+TIMEOUT = 540
```

Comment:
This is the same timeout defined for satellite scan, can both be removed and a constant be created on the constants file?

Reply:
This timeout is just what I chose based on trial and error for the vCenter, and I imagine may end up being different for Satellite, but I can't test it yet. I could make the name more informative and put it in the constants file, though. I'm thinking `NETWORK_SCAN_TIMEOUT`, `SATELLITE_SCAN_TIMEOUT` and `VCENTER_SCAN_TIMEOUT`

Reply:
if they are meant to be different then I think it is fine to differentiate but if not you can consider using the same name/constant for both. I mean I don't just want to add code that we can forget to optimize later (optimize in terms of maintainability)

Reply:
They are different at the moment because the localhost network type scans take significantly less time than the vcenter scans.